library_test.py

```diff
@@ -0,0 +1,120 @@
+"""Test script for midea-local library."""
+
+import asyncio
+import json
+import logging
+import sys
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+
+import aiohttp
+
+from midealocal.cloud import get_midea_cloud
+from midealocal.devices import device_selector
+from midealocal.discover import discover
+
+
+def get_arguments() -> tuple[ArgumentParser, Namespace]:
+    """Get parsed passed in arguments."""
+    parser = ArgumentParser(description="midea-local library test")
+    parser.add_argument(
+        "--username",
+        "-u",
+        type=str,
+        help="Set Cloud username",
+    )
+    parser.add_argument("--password", "-p", type=str, help="Set Cloud password")
+    parser.add_argument("--cloud_name", "-cn", type=str, help="Set Cloud name")
+    parser.add_argument(
```

Comment:
Add cloud options to help text
```suggestion
    parser.add_argument(
        "--cloud_name",
        "-cn",
        type=str,
        help="Set Cloud name, options are: " + ", ".join(clouds.keys()),
    )
```